app/views/download.scala.html

```diff
@@ -107,7 +108,15 @@ <h3 id="older-versions">Previous releases</h3>
       <table class="releases">
       @releases.previous.groupBy(_.version.slice(0, 3)).toSeq.sortBy(_._1).reverse.map { group =>
         <tbody class="version">
-          <tr><th colspan="3"><h4>@group._1</h4></th></tr>
+          <tr>
+            <th colspan="3">
+              <h4 class="previousVersionHeading">@group._1</h4>
+              @if(group._1 > "2.2"){
+                <a href="@{"https://www.playframework.com/documentation/" + group._1 + ".x/Installing"}" class="instructionLink">Instructions</a>
```

Comment:
Maybe we can call this "Setup Instructions" to be more clear.

Reply:
Yeah, you're right, I forgot this included 1.x versions.

Reply:
The current code also won't work if we ever have major versions greater than 10.

Reply:
yes, since we compare string. I'll update the changes

Reply:
yes, for 10.x.x versions it will fail from this line 
@releases.previous.groupBy(_.version.slice(0, 3))
I'll update it as well

Reply:
oh yes, due to the string comparison. I'll make the necessary changes